datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingService.java

```diff
@@ -587,8 +581,8 @@ public void disconnect() throws DataCapturingException {
      *
      * @throws IllegalStateException If communication with background service is not successful.
      */
-    @SuppressWarnings("WeakerAccess") // because we need to support this API - TODO really?
-    public void reconnect() throws DataCapturingException {
+    @SuppressWarnings("WeakerAccess") // TODO: not used by RS. Do we use it in our app?
```

Comment:
This should be used by all apps to rebind the app after it has been shut down.

Reply:
added this as TODO there so that I'll use this in our app and add sample code to the Readme